File: KremenchukZdo70Site.Domain/Response/CollectiveItemResponse.cs
namespace KremenchukZdo70Site.Domain.Response
{
    public class CollectiveItemResponse
    {
        public int Id { get; set; }
        public string? FirstName { get; set; }
        public string? SecondName { get; set; }
        public string? LastName { get; set; }
        public IEnumerable<string>? JobTitleNames { get; set; }
        public string? SmallProfileUrl { get; set; }
        public string? FullProfileUrl { get; set; }
    }
}

File: KremenchukZdo70Site/ClientApp/src/app/shared/models/collective-item-response.ts
import { JobTitleItemResponse } from './job-title-item-response';

export class CollectiveItemResponse {
  id: number = 0;
  firstName: string | undefined;
  secondName: string | undefined;
  lastName: string | undefined;
  smallProfileUrl: string | undefined;
  fullProfileUrl: string | undefined;
  jobTitles: JobTitleItemResponse[] | undefined;
}
